quickstart/docker/.env

```diff
@@ -11,7 +11,8 @@ ZITI_EDGE_ROUTER_ENROLLMENT_DURATION=
 ## Additional variables to override. 
 #ZITI_EDGE_CONTROLLER_RAWNAME=some.other.name.com
 #ZITI_EDGE_CTRL_ADVERTISED_HOST_PORT=some.other.name.com:1280
-#ZITI_CTRL_ADVERTISED_ADDRESS=some.other.name.com
 #ZITI_EDGE_CONTROLLER_HOSTNAME=some.other.name.com
-#ZITI_CONTROLLER_HOSTNAME=some.other.name.com
 #ZITI_EDGE_CONTROLLER_IP_OVERRIDE=20.20.20.20
+#ZITI_CTRL_ADVERTISED_ADDRESS=some.other.name.com
```

Comment:
just rearranged these and put 'edge controller' stuff near each other. one added line `ZITI_CTRL_PORT=6262` is referenced in the compose file now